dbms/src/Flash/Coprocessor/GenSchemaAndColumn.cpp

```diff
@@ -96,7 +95,8 @@ std::tuple<DM::ColumnDefinesPtr, size_t> genColumnDefinesForDisaggregatedRead(co
         const auto & column_info = table_scan.getColumns()[i];
         // Even if the id is pk_column or extra_table_id, we still output it as
         // a exchange receiver output column
-        const auto output_name = fmt::format("{}_{}", column_prefix, i);
+        const auto output_name = genNameForExchangeReceiver(i);
```

Comment:
```suggestion
        // Now the upper level seems treat disagg read as an ExchangeReceiver output, so
        // use this as output column prefix.
        const auto output_name = genNameForExchangeReceiver(i);
```

And remove line 88~89

Reply:
And remove line 88~89